include/rocksdb/options_type.h

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
```

Comment:
Can you temporarily move it outside include/rocksdb? We know that maybe eventually we'll make it public but we can move the file trivially at that time.